packages/next/server/config-shared.ts

```diff
@@ -414,6 +407,13 @@ export interface NextConfig extends Record<string, any> {
           exclude?: string[]
         }
     styledComponents?: boolean
+    emotion?:
```

Comment:
Might be good to add a warning that this has moved similar to https://github.com/vercel/next.js/blob/e2e7f04c9808af13e062559e1e1a90e2053b52a2/packages/next/server/config.ts#L432